build_config.py

```diff
@@ -6,9 +6,25 @@
 build_base_dir = 'bin'
 target_name = 'mip_upe_sdk'
 
-# Scons does not have built-in support to create multiple object files at the same time.
-# This helper fills that gap.
+INCLUDE_PATH_BASE = [
```

Comment:
Create a separate folder for all the build_ scripts. Like here https://github.com/AzureAD/mip-sdk-for-cpp/tree/develop/build